pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -302,10 +308,10 @@ func (facts *PipelineRunFacts) IsGracefullyStopped() bool {
 
 // DAGExecutionQueue returns a list of DAG tasks which needs to be scheduled next
 func (facts *PipelineRunFacts) DAGExecutionQueue() (PipelineRunState, error) {
-	tasks := PipelineRunState{}
-	// when pipeline run is stopping, gracefully cancelled or stopped, do not schedule any new task and only
+	var tasks PipelineRunState
```

Comment:
why changing to initialized variable to non-initialized?

Reply:
this is mostly personal preference. This change allows me to not have to include `want: PipelineRunState{}` in each test case in TestDAGExecutionQueue.

Reply:
Nice 👍